### DISCUSSION TITLE
2006/09/08/labview-%E7%9A%84%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87-2-%E7%A8%8B%E5%BA%8F%E6%85%A2%E5%9C%A8%E5%93%AA%E9%87%8C

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#113] [Announcements]
# 2006/09/08/labview-%E7%9A%84%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87-2-%E7%A8%8B%E5%BA%8F%E6%85%A2%E5%9C%A8%E5%93%AA%E9%87%8C

https://lv.qizhen.xyz/

https://blog.qizhen.xyz/2006/09/08/labview-%E7%9A%84%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87-2-%E7%A8%8B%E5%BA%8F%E6%85%A2%E5%9C%A8%E5%93%AA%E9%87%8C

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Qizhen
2009年04月14日 09:06 编辑
去除调试信息在VI的属性对话框里设置。最好使用Function Global代替Global Variable

回复
帆
2009年04月11日 17:29 编辑
Global Variable不用的话，不同VI传递数据是不是只依赖接线板？有别的途径？去除调试信息怎么实现呀？

回复
wings
2007年08月14日 19:08 编辑
呵呵,知道了,谢谢你,朋友 

回复
Qizhen
2006年09月11日 09:47 编辑
我既不是做黑客的，也不在美国。具体在哪，就不说了，免得你来揍我。
我把你的电话号码删掉了，公布自己的电话可能会有很多人来骚扰的。
 
我常去看别人的 blog，再从一个 blog 的连接调到另一个，看着看着就看到你的了 :)

回复
wings
2006年09月10日 16:33 编辑
前天我的电脑中了好多木马，害的我重装系统，是不是你弄的啊！！～～～？？说～～！我的本本在哭泣，哼，到现在我的内置摄像头还不能用，你以为你在美国就强啊，哼，不怕你！！小心我去美国找你去揍你哦～

回复
Qizhen
2006年09月9日 19:59 编辑
我也仅仅是早用了几年 LabVIEW 而已。如果你有关于 LabVIEW 的问题，我很愿意帮你研究的。你可以直接把问题张贴到这里，也可以给我写信：ruanqizhen@gmail.com。或者把问题发到论坛：
http://forums.ni.com/ni/board?board.id=2170
但是我的msn只在上班的时候使用，主要是用于工作需要的，与工作无关的问题一般就不用 msn 了。

回复
lei
2006年09月9日 13:12 编辑
hi,看了你关于Labview的文章知道你简直是一个高手，我现在正在做有关Labview的软件开发，但是我是属于那种初学者的程度，所以有很多的问题想向人请教，如果你不嫌弃收下我这个徒弟的话，能加我到你的MSN联系人中吗？这样可以有更多的机会和你交流啊！我的联系方式都写在这里，但是我找不到你的E-mail地址，所以方便的话你加加我，好吧！！不胜感谢！

回复
ea.amber.
2006年09月8日 22:42 编辑
收到一封来自你的邮件，本来以为又是病毒，可是后来发现居然可以打开，而且在里面看到了张芳的blog链接。所以，我想，你们应该是朋友；所以，我就路过打个招呼。hi～～
